### ISSUE TITLE
Table Sticky Header & Scrollbar bug

### ISSUE THREAD

**Issue by valllentinnaa** [SAP/ui5-webcomponents#1950]
**Describe the bug**

1. The table sticky header covers all that goes on top of it.
2. The scrollbar inside an UI5 component is not customizable. 

**To reproduce**
1. Use [Table](https://sap.github.io/ui5-webcomponents-react/?path=/story/ui5-web-components-table--generated-default-story) with _stickyColumnHeader;_
2. Use [Select](https://sap.github.io/ui5-webcomponents-react/?path=/docs/ui5-web-components-select--generated-default-story), [List](https://sap.github.io/ui5-webcomponents-react/?path=/docs/ui5-web-components-list--generated-default-story) or anything that invokes internal scrollbar.

**Expected behavior**
1. The table header should be with regular stacking context. 
2. The internal scrollbar should be customizable.

**Screenshots**
![image](https://user-images.githubusercontent.com/37929416/87402572-6fe8a180-c5c4-11ea-8a5d-7065b414259b.png)

**Context**
```"@ui5/webcomponents": "1.0.0-rc.7",
        "@ui5/webcomponents-base": "0.20.0",
        "@ui5/webcomponents-fiori": "1.0.0-rc.7",
        "@ui5/webcomponents-icons": "1.0.0-rc.7",
        "@ui5/webcomponents-react": "0.10.0-rc.7",
        "@ui5/webcomponents-react-base": "0.9.4",
        "@ui5/webcomponents-theme-base": "1.0.0-rc.7",```
```

**Comment by ilhan007** [comment]
Hello, could you provide a codesandbox sample or other sample, where we can look at the issue with table header overlapping popups, because it is not so easy to reproduce. For example I have a table with sticky headers, I have a Select that opens popup and as you can see the popup is over the Table header


<img width="995" alt="Screenshot 2020-07-15 at 11 29 04 AM" src="https://user-images.githubusercontent.com/15702139/87522637-8a874d00-c68e-11ea-927a-b82e25875143.png">

**Comment by valllentinnaa** [comment]
Sure, here you can check the table header:
[CodeSandBox](https://codesandbox.io/s/table-header-f3i3m?file=/App.js)

![image](https://user-images.githubusercontent.com/37929416/87854162-a4d45b80-c918-11ea-885c-c11982adfba0.png)

**Comment by ilhan007** [comment]
Hello @valllentinnaa  it turns out we already fixed that issue in the master, you can change all 1.0.0-rc.7 versions in the codesanbox example with 0.0.0-8b66a95c7 (our preview version) and see it yourself. The fix will be part of 1.0.0-rc.8, planned for the end of the next week.

Change versions:
<img width="437" alt="Screenshot 2020-07-24 at 3 39 22 PM" src="https://user-images.githubusercontent.com/15702139/88392089-16eaeb80-cdc4-11ea-9b9c-6b0ab05d5c59.png">

See the result:
<img width="532" alt="Screenshot 2020-07-24 at 3 39 06 PM" src="https://user-images.githubusercontent.com/15702139/88392056-09356600-cdc4-11ea-8283-e906bda3523e.png">

For the second thing, about the ScrollBar, you want to be able to customize it like this:

```css
::-webkit-scrollbar-track {
  background: #f1f1f1;
}

::-webkit-scrollbar-thumb {
  background: #888;
}
```

**Comment by ilhan007** [comment]
For the 1st issue
We released a new RC - 1.0.0-rc.8 and  "The table sticky header covers all that goes on top of it." is fixed.


For the 2nd issue 
if it still important for you please create a separate issue, explaining how do you expect to customize the scrollbar, perhaps like this:

```css
::-webkit-scrollbar-track {
  background: #f1f1f1;
}

::-webkit-scrollbar-thumb {
  background: #888;
}
```

BR,
ilhan
